src/app/routes/connector-ui/policy-definition-page/new-policy-dialog/new-policy-dialog.component.html

```diff
@@ -25,27 +25,57 @@ <h1 mat-dialog-title>Create New Policy</h1>
         </mat-option>
       </mat-select>
     </mat-form-field>
+    <!-- Time Interval Type -->
+    <ng-container *ngIf="form.policyType === 'Time-Period-Restricted'">
+      <mat-form-field color="accent">
+        <mat-label>Date Selection Criteria</mat-label>
+        <mat-select [formControl]="form.group.controls.dateSelectionType">
+          <mat-option value="Start-Date"> Start Date (open-ended) </mat-option>
+          <mat-option value="Date-Range"> Date Range</mat-option>
+        </mat-select>
+      </mat-form-field>
 
-    <!-- Date Range -->
-    <mat-form-field
-      *ngIf="
-        form.policyType == 'Time-Period-Restricted' &&
-          form.group.controls.range;
-        let ctrl
-      "
-      color="accent">
-      <mat-label>Time-Period</mat-label>
-      <mat-date-range-input [formGroup]="ctrl" [rangePicker]="picker">
-        <input matStartDate formControlName="start" placeholder="Start date" />
-        <input matEndDate formControlName="end" placeholder="End date" />
-      </mat-date-range-input>
-      <mat-hint>MM/DD/YYYY – MM/DD/YYYY</mat-hint>
-      <mat-datepicker-toggle matSuffix [for]="picker"></mat-datepicker-toggle>
-      <mat-date-range-picker #picker></mat-date-range-picker>
-      <mat-error *ngIf="ctrl.invalid">{{
-        validationMessages.invalidDateRangeMessage
-      }}</mat-error>
-    </mat-form-field>
+      <!-- Start Date  -->
+      <mat-form-field
+        *ngIf="
+          form.dateSelectionType == 'Start-Date' &&
+            form.group.controls.dateSelectionValue;
+          let ctrl
+        "
+        color="accent">
+        <mat-label>Choose a date</mat-label>
+        <input matInput [matDatepicker]="picker" [formControl]="ctrl" />
+        <mat-hint>MM/DD/YYYY</mat-hint>
+        <mat-datepicker-toggle matSuffix [for]="picker"></mat-datepicker-toggle>
+        <mat-datepicker #picker></mat-datepicker>
+        <mat-error *ngIf="ctrl.hasError('matDatepickerParse')">
+          {{ validationMessages.invalidDateMessage }}
+        </mat-error>
+      </mat-form-field>
+
+      <!-- Date Range -->
+      <mat-form-field
+        *ngIf="
+          form.dateSelectionType == 'Date-Range' && form.group.controls.range;
+          let ctrl
+        "
+        color="accent">
+        <mat-label>Choose a range</mat-label>
+        <mat-date-range-input [formGroup]="ctrl" [rangePicker]="picker">
+          <input
+            matStartDate
+            formControlName="start"
+            placeholder="Start date" />
+          <input matEndDate formControlName="end" placeholder="End date" />
+        </mat-date-range-input>
+        <mat-hint>MM/DD/YYYY – MM/DD/YYYY</mat-hint>
```

Comment:
Can you check if we can use a better date format for both fields?

Either YYYY-MM-DD or DD/MM/YYYY?

MM/DD/YYYY is the most confusing one